src/libprojectM/PipelineMerger.cpp

```diff
@@ -5,14 +5,6 @@
 const double PipelineMerger::e(2.71828182845904523536);
 const double PipelineMerger::s(0.5);
 
-void PipelineMerger::ensureAlphaIsNotBlended(const Pipeline & a)
-{
-	for ( std::vector<RenderItem*>::const_iterator pos = a.drawables.begin(); pos != a.drawables.end(); ++pos )
-	{
-		( *pos )->masterAlpha = 1.0;
```

Comment:
i don't really understand the error, so I don't know if a null ptr check here is sufficient or that's just hiding the real problem 
@psperl